packages/template-retail-react-app/app/hooks/use-active-data.js

```diff
@@ -76,18 +94,20 @@ const useActiveData = () => {
             }
         },
         async trackPage(siteId, localeId, currency) {
-            if (!canTrack()) return
+            if (!ACTIVE_DATA_ENABLED) {
+                return
+            }
             try {
+                await importDwActivities();
                 var activeDataUrl =
                     `${proxyBasePath}/ocapi/on/demandware.store/Sites-` +
                     siteId +
                     '-Site/' +
                     localeId +
                     '/__Analytics-Start'
-                var dwAnalytics = dw.__dwAnalytics.getTracker(activeDataUrl)
-                if (typeof dw.ac == 'undefined') {
-                    dwAnalytics.trackPageView()
```

Comment:
@SalesforceCommerceCloud/mobifyers That case was actually impossible to get into without removing all the analytics scripts. I changed it to execute the `trackPageView` method unconditionally, not sure if that's desired.